### DISCUSSION TITLE
Translation contribution

### DISCUSSION THREAD

**Discussion by yoojincha** [yangshun/tech-interview-handbook#263] [General]
Hi, I wanted to discuss before making a non-trivial (I think) contribution :)
I would like to translate some of these .md files into Korean if you're up to having translations !
Please let me know what you think about this. Thanks🙌

**Comment by toothsy** [comment]
Sorry yangshun but I'm not familiar with Korean language

**Reply by JCluzet** [reply]
Maybe doing basic translation before? like english or other

**Comment by toothsy** [comment]
The only languages I know are English, Hindi Jcluzet so I doubt i can be of
much help...
Sorry
